public/geojson/areas/spain_autonomous_communities.geojson

```diff
@@ -0,0 +1 @@
[-13.620752081635647,29.12831360375291],[-13.603017929610786,29.13667156590776],[-13.585283777585925,29.13478428413086],[-13.562746626054333,29.116181078044253],[-13.548337627534133,29.11779874813874],[-13.524692091500985,29.14179418787364],[-13.523214245498913,29.165520015926127],[-13.513608246485447,29.177922153317198],[-13.511022015981823,29.195446912674146],[-13.497721401963176,29.21782468231456],[-13.472598019927958,29.233192548212195],[-13.469642327923815,29.241550510367045]]],[[[-13.50806632397768,29.291159059931335],[-13.482573480441943,29.280104980952334],[-13.471859096926922,29.26932051365575],[-13.487745941449193,29.25395264775812],[-13.488484864450228,29.243977015508776],[-13.50400224747198,29.224295362692512],[-13.521736399496842,29.21782468231456],[-13.544273551028436,29.218094293996973],[-13.546490320031545,29.232653324847362],[-13.53282024451238,29.2388543935429],[-13.521366937996325,29.257457599629507],[-13.524692091500985,29.267163620196435],[-13.50806632397768,29.291159059931335]]],[[[-13.50400224747198,29.41733732730137],[-13.488854325950747,29.402239073086154],[-13.50621901647509,29.383635866999548],[-13.530234014008755,29.38633198382369],[-13.530603475509272,29.40250868476857],[-13.50400224747198,29.41733732730137]]],[[[-15.861166620776377,30.15526450207016],[-15.854516313767054,30.13773974271321],[-15.87483669629554,30.14043585953736],[-15.871142081290362,30.154186055340503],[-15.861166620776377,30.15526450207016]]]]}},{"type":"Feature","properties":{"ISO":"ESP","NAME_0":"Spain","ID_1":949,"NAME_1":"La Rioja","VARNAME_1":"Rioja","HASC_1":"ES.LO","TYPE_1":"Comunidad Autónoma","ENGTYPE_1":"Autonomous Community"},"geometry":{"type":"Polygon","coordinates":[[[-2.8579691100478186,42.63828539786664],[-2.8428211885265835,42.62965782402937],[-2.841712804025029,42.60862811280103],[-2.8199145754944723,42.614559569814155],[-2.8221313444975795,42.58894645998477],[-2.8406044195234763,42.58166694455957],[-2.8376487275193334,42.5711520889454],[-2.8173283449908464,42.55767150482467],[-2.798116346963914,42.57384820576954],[-2.780382194939053,42.57951005110025],[-2.771884580427141,42.59622597540996],[-2.7737318879297312,42.60701044270654],[-2.7637564274157462,42.62291753196901],[-2.730504892369133,42.617525298320714],[-2.7061204333349487,42.59973092728135],[-2.6806275897992116,42.594608305315475],[-2.6861695123069804,42.58301500297164],[-2.6913419733142323,42.54284286229186],[-2.705381510333913,42.516690529097644],[-2.6854305893059447,42.520195480969036],[-2.67730243629455,42.5102198487197],[-2.656612592265546,42.50401878002416],[-2.6499622852562243,42.48541557393755],[-2.6285335182261846,42.49566081786931],[-2.592326291175427,42.4918862543155],[-2.5779172926552274,42.482989068795824],[-2.537276527598255,42.48325868047824],[-2.538015450599291,42.49404314777482],[-2.519542375573394,42.508062955260385],[-2.514739376066661,42.4899989725386],[-2.4770543030138317,42.48676363234963],[-2.4489752289744704,42.497008876281384],[-2.4386303069599684,42.
```

Comment:
Unfortunately, we have an issue in this geojson, some "provincia" values don't match with OpenStreetMap :

1. Received from openstreetmap "Navarra" but "Comunidad Foral de Navarra"  in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=42.63008462628674&lon=-1.6433664720849774&zoom=5&addressdetails=1&accept-language=es
2. Received from openstreetmap "Asturias" but "Principado de Asturias" in geojson
Tested with https://nominatim.openstreetmap.org/reverse?format=json&lat=43.27224333834266&lon=-5.946760248088729&zoom=5&addressdetails=1&accept-language=es